### PR TITLE
Add article on progress update

### PR DESCRIPTION
**Pull request by cburgdorf**

### CODE REVIEW

File: source/_posts/progress-update-on-our-machine-learning-jump-start-course.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Progress update on our Machine Learning Jump Start course
+date: 2018-03-21 12:00:00
+tags: announcement, course
+---
+
+Last year we [announced to build a fresh, beginner-friendly Machine Learning course](/2017/12/15/machine-learning-jump-start-online-course/) with the promise to make this exciting field more accesible for starters. Many people want to get started with Machine Learning but have yet to find a course that isn't overloaded by heavy math and science. Our Machine Learning Jump Start breaks with that by explaing concepts in simple terms and focusing on easy to follow, practical online labs.
```
Comment (PascalPrecht) [inline source/_posts/progress-update-on-our-machine-learning-jump-start-course.md]: > [...] exciting field more acces**s**ible for [...]

> Our [Machine Learning Jump Start](https://course.machinelabs.ai) breaks [...]

> [...] breaks with that by explain**in**g concepts [...]

> [...] practical online labs **using the MachineLabs platform**

File: source/_posts/progress-update-on-our-machine-learning-jump-start-course.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+<!-- more -->
+
+The course has been in [pre-sale](http://course.machinelabs.ai) since then and it's been very motivating for us to see all the excitment about the course from people all around the world.
```
Comment (PascalPrecht) [inline source/_posts/progress-update-on-our-machine-learning-jump-start-course.md]: How about:

> [...] and it's been very motivating for us to see all the excit**e**ment from our community all around the world.

File: source/_posts/progress-update-on-our-machine-learning-jump-start-course.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+The format of the course is mainly video / screencast driven but also includes interactive labs that run entirely on the MachineLabs platform.
```
Comment (PascalPrecht) [inline source/_posts/progress-update-on-our-machine-learning-jump-start-course.md]: Not sure what exactly is the purpose of this sentence here.

File: source/_posts/progress-update-on-our-machine-learning-jump-start-course.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Today we like to share an update about our progress and the current timeline for the course.
+
+Before we dive into this, enjoy this little teaser video that we recorded for you at our last company offsite in Oslo earlier this year.
```
Comment (PascalPrecht) [inline source/_posts/progress-update-on-our-machine-learning-jump-start-course.md]: Merge this paragraph into the previous one.

File: source/_posts/progress-update-on-our-machine-learning-jump-start-course.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+INTRO_VIDEO_HERE
+
+While we consider the course a long-term effort and plan to update the courseware continuously, our initial plan was to get the Early Access Preview (EAP) out this month.
```
Comment (PascalPrecht) [inline source/_posts/progress-update-on-our-machine-learning-jump-start-course.md]: >  [...] and plan to update the <s>courseware</s> it continuously,

> [...] get the Early Access Preview (EAP) out this month (April 2018).

^ This last one is mainly for people who'll read the post much later and don't pay attention to the publish date of the post.

File: source/_posts/progress-update-on-our-machine-learning-jump-start-course.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+**Please note that no one has been or will be charged a single dollar before the EAP is out.**
+
+That, plus the fact that we give a **100 % money back guarantee** for everyone who doesn't like the course, makes the pre-order a steal deal. It basically means everyone can pre-order the course now at a reduced rate, doesn't pay anything before the EAP ships and has the safety net of not paying anything if the course doesn't match the expectations.
+
+To ensure we're shipping something that people will like we've decided to take a bit more time for our EAP to mature.
+
+Our new target date to deliver the EAP is the **end of Q3 2018**. Once the EAP is out everyone can expect to receive **monthly updates** to the courseware.
```
Comment (PascalPrecht) [inline source/_posts/progress-update-on-our-machine-learning-jump-start-course.md]: Merge this paragraph into the previous one.

File: source/_posts/progress-update-on-our-machine-learning-jump-start-course.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+Today we like to share an update about our progress and the current timeline for the course.
+
+Before we dive into this, enjoy this little teaser video that we recorded for you at our last company offsite in Oslo earlier this year.
+
+INTRO_VIDEO_HERE
+
+While we consider the course a long-term effort and plan to update the courseware continuously, our initial plan was to get the Early Access Preview (EAP) out this month.
+
+**Please note that no one has been or will be charged a single dollar before the EAP is out.**
```
Comment (PascalPrecht) [inline source/_posts/progress-update-on-our-machine-learning-jump-start-course.md]: > Please <s>note</s> remember that no one has or will [...]

This and the following paragraph should be moved to *after* we say "we're pushing it back"

**Comment by cburgdorf** [comment]
@PascalPrecht Thanks for the feedback. Addressed all points. Maybe our youtube channel administrator @d3lm could upload the intro video so that I can include it in the post.

**Comment by d3lm** [comment]
Done. Video is uploaded.

**Pull request merged**